src/Http/Controllers/SwaggerController.php

```diff
@@ -33,8 +38,15 @@ public function docs($jsonFile = null)
 
         $content = File::get($filePath);
 
+        $contentType = 'application/json';
+        if ($extension === 'yaml') {
+            // Use text/plain instead of application/yaml to prevent triggering a file
+            // download in Firefox
+            $contentType = 'text/plain';
```

Comment:
Maybe we can use `application/yaml` content type and add `Content-Disposition: inline` header instead?

Reply:
I'm ok with this but it still causes a file download and I was looking to just have the yaml displayed in plain-text in the browser.

Reply:
But `Content-Disposition: inline` should tel browser NOT to download the file. Did you tested this? If this is still downloading maybe where is other headers which can force browser to show file instead of download?